worker/include/RTC/TransportCongestionControlServer.hpp

```diff
@@ -89,6 +92,9 @@ namespace RTC
 		uint8_t unlimitedRembCounter{ 0u };
 		std::deque<double> packetLossHistory;
 		double packetLoss{ 0 };
+		bool receivedTransportWideSeqNumber{ false };
+		uint16_t transportCcFeedbackStartSeqNum{ 0u };
```

Comment:
perhaps this is better?

```suggestion
		uint16_t transportCcFeedbackWideSeqNumStart{ 0u };
```